### DISCUSSION TITLE
Impianto BTicino Living Now nuovo, non smart, da smartizzare

### DISCUSSION THREAD

**Discussion by filippomirandola** [indomus/forum#2522] [Hardware & Firmware]
Ciao a tutti!
La casa nuova dei miei genitori, in costruzione, si sta ultimando e vorrei aiutarli a renderla più smart.
Premesse: uno o due anni fa avevo già spulciato approfonditamente il blog e il forum, per cui le nozioni di base più o meno le conosco, ma vorrei da voi un aiuto più specifico, anche alla luce delle recenti innovazioni riguardo a Matter.
In particolare, nella casa sono state installate delle placche BTicino Living Now tradizionali (**non smart**), mentre sono smart i controlli del videocitofono e del riscaldamento/raffrescamento: proprio oggi ho visto che è stato posizionato lo [Smarther2](https://www.bticino.it/termostati/smarther-termostato-wifi) di BTicino/Netatmo e [questo videocitofono](https://www.bticino.it/citofoni-videocitofoni/classe100-videocitofono-wi-fi). Credo non sia stato installato il gateway di BTicino (a meno che non sia necessario per lo Smarther, onestamente non lo so).

Mio padre si era consultato con un amico che lavorava nell'ambito che gli aveva consigliato di aspettare a prendere le prese smart, comprandole solo quando avrebbe capito di quali avrebbe avuto effettivamente bisogno.

Ovviamente sposo la vostra filosofia e preferisco un sistema HomeAssistant rispetto a quello chiuso di BTicino, per motivi che conoscete meglio di me. Voi, al mio posto, che fareste? Comprereste il giusto numero di shelly/sonoff da installare dove servono (follow-up question: si possono installare gli shelly nelle placche living now, vero?) o è comunque meglio comprare le placche smart LivingNow? Usando HomeAssistant rischio di far danni al sistema smart del termostato o è pienamente compatibile? Mi confermate che ora conviene orientarsi su un minipc e non sul raspberry? 

Grazie mille in anticipo. Se ci sono state discussioni simili mi scuso e vi chiedo gentilmente di linkarmele :)
Scusatemi in anticipo se alcune domande sono formulate male

**Comment by essegi65** [comment]
Le prese e interruttori Bticino living now puoi integrarli all'interno di HA tranquillamente (sia usando il loro GW che senza tramite Z2M) e se non hai problemi di spesa credo siano la miglior soluzione (non modifichi impianto elettrico, non aggiungi niente all'interno delle scatole 503 a volte gli spazi sono veramente ridotti, se togli HA non devi modificare niente per tornare all'impianto originale, ecc.)
Al contrario se vuoi contenere i costi puoi dirottare la scelta su prodotti wi-fi (sonoff, shelly, ecc.) o Z2M (sonoff, tuya, aqara, ecc.) ma dovrai avere dimestichezza con gli impianti elettrici o meglio ancora li dovrai far installare da un elettricista.
Ambe due le soluzioni sono espandibili sia da subito che in futuro con valvole termostatiche, termostati, sensori movimento, ecc.
Io uso sia prodotti living now che shelly e sonoff.

**Reply by filippomirandola** [reply]
Grazie mille per il consiglio. Ho trovato un'offerta buona per il sistema BTicino, intanto installiamo quello :) Gentilissimo!

**Reply by wit3** [reply]
Scusate, mi accodo a questa discussion perchè ho il medesimo dubbio/problema.
Ho una casa in ritrutturazione e l'elettricista mi ha già praticamente ultimato il sistema Bticino Living Now tradizionale ( quello cablato con MyHome).
Ora la mia domanda è una, mantenendo questo impianto, posso in futuro( nei prossimi mesi ) interfacciarmi con luci/tapparelle e/o tutte le possibli combinazioni offerte da HA ?.

Oppure finchè sono in tempo ( ma con un costo maggiore dato dalla mia confusione ) mi conviene fargli sostituire l'impianto Living Now MyHome con uno living now Smart?

Grazie mille, sia a @filippomirandola che a @essegi65 per lo spunto di riflessione.

**Comment by Fentadroid** [comment]
> Ciao a tutti! La casa nuova dei miei genitori, in costruzione, si sta ultimando e vorrei aiutarli a renderla più smart. Premesse: uno o due anni fa avevo già spulciato approfonditamente il blog e il forum, per cui le nozioni di base più o meno le conosco, ma vorrei da voi un aiuto più specifico, anche alla luce delle recenti innovazioni riguardo a Matter. In particolare, nella casa sono state installate delle placche BTicino Living Now tradizionali (**non smart**), mentre sono smart i controlli del videocitofono e del riscaldamento/raffrescamento: proprio oggi ho visto che è stato posizionato lo [Smarther2](https://www.bticino.it/termostati/smarther-termostato-wifi) di BTicino/Netatmo e [questo videocitofono](https://www.bticino.it/citofoni-videocitofoni/classe100-videocitofono-wi-fi). Credo non sia stato installato il gateway di BTicino (a meno che non sia necessario per lo Smarther, onestamente non lo so).
> 
> Mio padre si era consultato con un amico che lavorava nell'ambito che gli aveva consigliato di aspettare a prendere le prese smart, comprandole solo quando avrebbe capito di quali avrebbe avuto effettivamente bisogno.
> 
> Ovviamente sposo la vostra filosofia e preferisco un sistema HomeAssistant rispetto a quello chiuso di BTicino, per motivi che conoscete meglio di me. Voi, al mio posto, che fareste? Comprereste il giusto numero di shelly/sonoff da installare dove servono (follow-up question: si possono installare gli shelly nelle placche living now, vero?) o è comunque meglio comprare le placche smart LivingNow? Usando HomeAssistant rischio di far danni al sistema smart del termostato o è pienamente compatibile? Mi confermate che ora conviene orientarsi su un minipc e non sul raspberry?
> 
> Grazie mille in anticipo. Se ci sono state discussioni simili mi scuso e vi chiedo gentilmente di linkarmele :) Scusatemi in anticipo se alcune domande sono formulate male

Che tipo di impianto di raffrescamento utilizzano i tuoi genitori? A pavimento o climatizzatori?

**Comment by mgritti83** [comment]
Ciao,
Qualcuno può aiutarmi? ho un impianto bitcino living now, vorrei gestirlo da un pannello touch con Apple home e’ possibile?
Grazie
